fuzzers/074-dump_all/jobtiles.tcl

```diff
@@ -27,26 +27,6 @@ set not_allowed_tiles [get_tiles -of_objects $not_allowed_sites]
 set MAGIC 0.6875
 
 set speed_model_index_map [dict create]
```

Comment:
Lines 27-29 should also be deleted?

Reply:
Comment on line 26 didn't make it to the new file

Reply:
Done

Reply:
@litghost Just to know, where does this magic value originally comes from?

Reply:
Empirical measurement, hence why I called it "MAGIC".  It is a constant that seems to generate valid values, but I don't really know what it is.  Hence the comment about converting from ??? to Ohm like values.